mailchimp3/helpers.py

```diff
@@ -58,7 +58,7 @@ def check_email(email):
     :type email: :py:class:`str`
     :return: Nothing
     """
-    if not re.match(r"(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)", email):
+    if not re.match(r"(^[a-zA-Z0-9_.+'-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)", email):
```

Comment:
Please add information about the change to the docstring as mentioned in the pull request comments. Countries like Ireland that have apostrophes in their names use them in email and it is valid per the email spec. Once that is updated, we can merge this in.